.github/workflows/dev-deploy.yml

```diff
@@ -0,0 +1,174 @@
+name: Development Stage
+
+on:
+  workflow_run:
+    workflows: [Unit Test]
+    types: [completed]
+
+env:
+  SAM_TEMPLATE: template.yaml
+  TESTING_STACK_NAME: pdf-encrypt-dev
+  TESTING_REGION: ap-northeast-1
+  TESTING_PIPELINE_EXECUTION_ROLE: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/${{ secrets.TESTING_PIPELINE_EXECUTION_ROLE_NAME }}
+  TESTING_CLOUDFORMATION_EXECUTION_ROLE: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/${{ secrets.TESTING_CLOUDFORMATION_EXECUTION_ROLE_NAME }}
+
+permissions:
+  id-token: write
+  contents: read
+jobs:
+
+  give-policy-access:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      #- name: install-aws-cli
+      #  uses: unfor19/install-aws-cli-action@v1
+
+      - name: Assume the CloudFormation role
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          aws-region: ${{ env.TESTING_REGION }}
+          role-to-assume: ${{ env.TESTING_CLOUDFORMATION_EXECUTION_ROLE }}
```

Comment:
@javanlth this is a secret

Reply:
I checked, env variables constructed with secrets still mask the secrets.